src/app.jsx

```diff
@@ -439,12 +439,14 @@ const ForceDeleteModal = ({ selected, itemPath, errorMessage }) => {
 
     const forceDelete = () => {
         cockpit.spawn(["rm", "-rf", itemPath], { superuser: "try" })
-                .then(Dialogs.close, (err) => { Dialogs.show(<DeleteErrorModal errorMessage={err.message} />) });
+                .then(Dialogs.close, (err) => { Dialogs.show(<ForceDeleteModal selected={undefined} itemPath={undefined} errorMessage={err.message} />) });
```

Comment:
It's better to pass the existing itemPath and selected so that you use the same title.